### PR TITLE
[bitnami/grafana-operator] Add support for mounting extra files into the Grafana Pod

### PR DESCRIPTION
**Pull request by vigohe**
**Description of the change**

This PR will add support for Mounting extra files into the Grafana Pod that is missing in the helm chart.
ref: https://github.com/integr8ly/grafana-operator/blob/master/documentation/extra_files.md

**Benefits**

Support for Mounting extra files into the Grafana Pod 

**Possible drawbacks**

None

**Applicable issues**

N/A

**Additional information**

N/A

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

### CODE REVIEW

File: bitnami/grafana-operator/values.yaml [hunk 1]
```diff
@@ -418,6 +418,13 @@ grafana:
       #   # allowed_domains: example.com
       #   allow_sign_up: True
 
+  ## Mounting extra files into the Grafana Pod
+  ## Ref: https://github.com/integr8ly/grafana-operator/blob/master/documentation/extra_files.md
+  ##
+  confiMaps: {}
```
Comment (alemorcuq) [inline bitnami/grafana-operator/values.yaml:424]:
```suggestion
  configMaps: []
```

As it is an array. Typo as well.

File: bitnami/grafana-operator/values.yaml [hunk 2]
```diff
@@ -418,6 +418,13 @@ grafana:
...
+  # - ldap-config
+  secrets: {}
```
Comment (alemorcuq) [inline bitnami/grafana-operator/values.yaml:426]:
```suggestion
  secrets: []
  #  - myCustomSecret
  #
```


As it is an array.

File: bitnami/grafana-operator/values.yaml [hunk 3]
```diff
@@ -418,6 +418,13 @@ grafana:
       #   # allowed_domains: example.com
       #   allow_sign_up: True
 
+  ## Mounting extra files into the Grafana Pod
```
Comment (alemorcuq) [inline bitnami/grafana-operator/values.yaml:421]:
```suggestion
  ## Mount extra configmaps and secrets into the Grafana pod
```

File: bitnami/grafana-operator/values.yaml [hunk 4]
```diff
@@ -418,6 +418,13 @@ grafana:
...
+  ## Ref: https://github.com/integr8ly/grafana-operator/blob/master/documentation/extra_files.md
+  ##
+  confiMaps: {}
+  # - ldap-config
```
Comment (alemorcuq) [inline bitnami/grafana-operator/values.yaml:425]:
```suggestion
  #  - myCustomConfigMap
  #
```

**Review by alemorcuq — CHANGES_REQUESTED** [review]
Thanks for the PR! I've added some suggestions, please take a look:

File: bitnami/grafana-operator/values.yaml (hunk 1 above)
Comment (vigohe) [reply bitnami/grafana-operator/values.yaml:424]: @alemorcuq  but if I set it as array will this code work ??

```
{{ toYaml .Values.grafana.configMaps | nindent 4 }}
```

Comment (vigohe) [reply bitnami/grafana-operator/values.yaml:424]: Did work! I thought It was necessary to use range when using arrays.

File: bitnami/grafana-operator/templates/grafana.yaml [hunk 5]
```diff
@@ -131,6 +131,12 @@ spec:
     {{- end }}
     path: {{ include "common.tplvalues.render" (dict "value" .Values.grafana.ingress.path "context" $) }}
   config: {{- include "common.tplvalues.render" ( dict "value" .Values.grafana.config "context" $ ) | nindent 4 }}
+  {{- if .Values.grafana.configMaps }}
+  configMaps: {{ toYaml .Values.grafana.configMaps | nindent 4 }}
+  {{- end }}
+  {{- if .Values.grafana.configMaps }}
```
Comment (alemorcuq) [inline bitnami/grafana-operator/templates/grafana.yaml:137]:
```suggestion
  {{- if .Values.grafana.secrets }}
```

**Review by alemorcuq — CHANGES_REQUESTED** [review]
It looks that there's a conflict with the Chart version. Can you check that? I think you will need to rebase your branch.

**Comment by vigohe** [comment]
> It looks that there's a conflict with the Chart version. Can you check that? I think you will need to rebase your branch.

@alemorcuq now it's ok!

**Review by alemorcuq — APPROVED** [review]
Great, thanks!

**Pull request merged**